app/models/user.rb

```diff
@@ -1,10 +1,15 @@
 class User < ApplicationRecord
-  has_many :posts, foreign_key: :author_id
-  has_many :comments
-  has_many :likes
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] As pointed out by the previous reviewer, please kindly include `:confirmable` as part of the devise modules.